src/Controls/src/Core/Shell/Shell.cs

```diff
@@ -94,77 +121,215 @@ static void OnFlyoutItemIsVisibleChanged(BindableObject bindable, object oldValu
+		/// </summary>
+		/// <param name="obj">The object that that gets the navigation bar visibility.</param>
+		/// <returns><see langword="true"/> if the navigation bar is visible; otherwise, <see langword="false"/>.</returns>
 		public static bool GetNavBarIsVisible(BindableObject obj) => (bool)obj.GetValue(NavBarIsVisibleProperty);
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='SetNavBarIsVisible']/Docs/*" />
+
+		/// <summary>
+		/// Controls if the navigation bar is visible when a page is presented. 
+		/// By default the value of the property is <see langword="true"/>.
+		/// </summary>
+		/// <param name="obj">The object that modifies the navigation bar visibility.</param>
+		/// <param name="value"><see langword="true"/> to set the navigation bar as visible; otherwise, <see langword="false"/>.</param>
 		public static void SetNavBarIsVisible(BindableObject obj, bool value) => obj.SetValue(NavBarIsVisibleProperty, value);
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetNavBarHasShadow']/Docs/*" />
+		/// <summary>
+		/// Gets a value that represents if the navigation bar has a shadow.
+		/// </summary>
+		/// <param name="obj">The object that modifies if the navigation bar has a shadow.</param>
+		/// <returns>Returns if the navigation bar has a shadow.</returns>
 		public static bool GetNavBarHasShadow(BindableObject obj) => (bool)obj.GetValue(NavBarHasShadowProperty);
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='SetNavBarHasShadow']/Docs/*" />
+
+		/// <summary>
+		/// Controls whether the navigation bar has a shadow. 
+		/// By default the value of the property is <see langword="true"/> on Android, and <see langword="false"/> on other platforms.
+		/// </summary>
+		/// <param name="obj">The object that modifies if the navigation bar has a shadow.</param>
+		/// <param name="value">Manages if the navigation bar has a shadow.</param>
 		public static void SetNavBarHasShadow(BindableObject obj, bool value) => obj.SetValue(NavBarHasShadowProperty, value);
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetSearchHandler']/Docs/*" />
+		/// <summary>
+		/// Gets the integrated search functionality.
+		/// </summary>
+		/// <param name="obj">The object that modifies the Shell search functionality.</param>
+		/// <returns>The integrated search functionality.</returns>
 		public static SearchHandler GetSearchHandler(BindableObject obj) => (SearchHandler)obj.GetValue(SearchHandlerProperty);
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='SetSearchHandler']/Docs/*" />
+
+		/// <summary>
+		/// Sets the handler responsible for implementing the integrated search functionality.
+		/// Enabling it results in a search box being added at the top of the page.
```

Comment:
```suggestion
		/// Enabling this property results in a search box being added at the top of the page.
```

I would add this to remarks